ci/build_docs.sh

```diff
@@ -36,7 +36,6 @@ rapids-mamba-retry install \
   --channel "${RAPIDS_LOCAL_CONDA_CHANNEL}" \
   --channel legate \
   --channel legate/label/experimental \
-  --channel legate/label/rc \
```

Comment:
Should we just leave this?  I doubt it hurts, and then we don't have to add it again in the future.

Reply:
I think it'd be safe to leave, and preferable to moving this in and out based on where you are in development.

Alternatively, for these libraries we could do something like RAPIDS libraries do, where they have code that selectively adds / removes channels based on whether a build is a PR or a release:

https://github.com/rapidsai/gha-tools/blob/main/tools/rapids-configure-conda-channels